src/test/client.rs

```diff
@@ -602,3 +602,49 @@ async fn x509_auth() {
         .await
         .unwrap();
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+#[function_name::named]
+async fn plain_auth() {
+    let _guard: RwLockReadGuard<_> = LOCK.run_concurrently().await;
```

Comment:
This test is based on the "Testing LDAP" page in our wiki. It uses the credentials listed as being safe to share externally. However, since the domain we connect to is only reachable from within the office or the VPN, the test skips unless a specific environmental variable is set.